tests/conftest.py

```diff
@@ -62,6 +63,26 @@ def increment_cache_key_prefix():
     cache.key_prefix = "t{0}".format(key_prefix_counter)
 
 
+@pytest.fixture(scope="session", autouse=True)
+def block_unmocked_requests():
```

Comment:
Cripes, my sincere apologies. I skimmed over the comment, which was dumb of me. I can't quite explain why... As always: you rocked it! :)